packages/make-styles/src/makeStyles.test.ts

```diff
@@ -49,6 +50,39 @@ describe('makeStyles', () => {
     `);
   });
 
+  it('handles RTL for styles', () => {
+    const computeClasses = makeStyles({
+      root: {
+        paddingLeft: '10px',
+        borderLeftWidth: '10px',
+      },
+    });
+
+    const ltrClasses = computeClasses({ dir: 'ltr', renderer, tokens: {} }).root;
+    const rtlClasses = computeClasses({ dir: 'rtl', renderer, tokens: {} }).root;
+
+    expect(ltrClasses).toEqual('__947mlk0 frdkuqy0 f1c8chgj');
+    expect(rtlClasses).toEqual('rtl __947mlk0 frdkuqy0 f1c8chgj');
+
+    // Classes should be the same for LTR & RTL
+    expect(`${RTL_CLASSNAME} ${ltrClasses}`).toEqual(rtlClasses);
+
+    expect(getCSSRules(renderer.styleElement)).toMatchInlineSnapshot(`
```

Comment:
Is it correct that rtl classes will always be rendered but only applied by `ax` ?